modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java

```diff
@@ -253,8 +253,21 @@ private void _validate(
 		}
 
 		if (Objects.equals(
+				objectActionTriggerKey,
+				ObjectActionTriggerConstants.KEY_ON_AFTER_DELETE) &&
+			Objects.equals(
+				objectActionExecutorKey,
+				ObjectActionExecutorConstants.KEY_UPDATE_OBJECT_ENTRY)) {
+
+			throw new UnsupportedOperationException();
```

Comment:
We should throw an exception specific to the Object Action. This exception is too generic. What do you think?